scrooge-sbt-plugin/src/main/scala/com/twitter/ScroogeSBT.scala

```diff
@@ -106,20 +110,17 @@ object ScroogeSBT extends Plugin {
    */
   val genThriftSettings: Seq[Setting[_]] = Seq(
     scroogeBuildOptions := Seq("--finagle"),
-    scroogeThriftSourceFolder <<= (sourceDirectory) { _ / "thrift" },
+    scroogePublishThrift := false,
+    scroogeThriftSourceFolder <<= (sourceDirectory in Compile) { _ / "thrift" },
     scroogeThriftExternalSourceFolder <<= (target) { _ / "thrift_external" },
     scroogeThriftOutputFolder <<= (sourceManaged) { identity },
     scroogeThriftIncludeFolders <<= (scroogeThriftSourceFolder) { Seq(_) },
     scroogeThriftNamespaceMap := Map(),
     scroogeThriftDependencies := Seq(),
+    libraryDependencies += "com.twitter" %% "scrooge-core" % com.twitter.BuildInfo.version,
```

Comment:
Before, a user needed to know that if they were using Scrooge through the sbt plugin that they also needed to add `scrooge-core` to their library dependencies. Now, this knowledge is not required, the plugin will add the dependency on it's own since I can't think of a reasonable way to be using Scrooge but not require `scrooge-core` to be in the classpath.

It's still possible for someone to override that version if for some reason they prefer not to use the same version of scrooge-core and sbt-plugin even though that is playing with fire to a certain extent.

This is a somewhat orthogonal change that sneaked in even though I think it makes sense.